### DISCUSSION TITLE
Cannot display pictures after code block in README

### DISCUSSION THREAD

**Discussion by CSA** [community/community#22243] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,<br>
If I use <code>&lt;/br&gt;</code> to wrap the line after a code block, the following picture will not be displayed.<br>
<a href="https://github.com/ftp/Test/blob/d7749b81c65b152b7a223ae1253cb1799912c776/README.md" rel="noopener nofollow ugc">CorrectReadMe</a><br>
<a href="https://github.com/ftp/Test/blob/d7749b81c65b152b7a223ae1253cb1799912c776/README_Test.md" rel="noopener nofollow ugc">IncorrectReadMe</a></p>
<p>Thanks</p>

**Comment by darkRaspberry** [comment, marked as answer]
<p>Just add a <code>enter key</code> after <code>&lt;br&gt;</code> line ends. It solves the issue.<br>
like this…</p>
<pre><code class="lang-auto">%Ramdom text/pic% &lt;br&gt;

[]() ... ramdom text and video
</code></pre>
<p>HINT- ENTER KEY Is big key near numeric pad in physical or virtual keyboad.</p>

**Comment by CSA** [comment]
<p>I try it in my project, it works, Thanks :slight_smile:</p>
